packages/connect/src/connector/specs/generated/polkadot.ts

```diff
@@ -0,0 +1 @@
+export {}
```

Comment:
What is this?

Reply:
Yeah... So, we currently have this "hacky" script named [`generate-specs`](https://github.com/paritytech/substrate-connect/blob/master/packages/connect/package.json#L28) which what it does is after the `build` has finished it adds a js file with the stringified version of each well known chain-spec into the `dist` folder... So, basically, it's like if we had those files before the build.

What happens, though, is that I have a test that checks that the `addWellKnownChain` API actually imports the chainspec of its corresponding file, in order to do that I have to mock that import, but when I try to mock it `jest` complains saying that the file doesn't exist (because it doesn't, it will be created after build, but that shouldn't be the case). So, in order to work around that I've put an empty file. In the future I'm planing to improve this and make sure that the [`checkSpecs` script](https://github.com/paritytech/substrate-connect/blob/master/packages/connect/package.json#L22) (not to be confused with the `generate-specs`) will add the ts files before build with the stringified content of the well known specs, that way we won't need to have this silly file so that jest doesn't crash.

Alternatively, though, if you want I can disable that test and remove this file until that work is done.

Reply:
Yeah I'd actually prefer to remove that test.